dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -102,10 +102,13 @@ function accept()
    })
    misn.markerAdd( mem.dest_planet )
    hook.land( "land" )
-   hook.enter( "pirate_ambush" )
    local colonel_ship = ship.get("Dvaered Arsenal")
    escort.init ( colonel_ship, {
    })
+   
+   if not naev.claimTest( system.cur(), true ) then return end
```

Comment:
Sorry, I meant it should be at the top of the `function pirate_ambush()` function.